packages/client/hmi-client/src/workflow/ops/model-coupling/tera-model-coupling.vue

```diff
@@ -223,13 +215,34 @@ const runCodeModelCoupling = () => {
 		});
 };
 
-onMounted(() => {
-	initialize();
-});
-
 onUnmounted(() => {
 	kernelManager.shutdown();
 });
+
+watch(
+	() => props.node.inputs,
+	async () => {
+		const inputs = props.node.inputs;
+
+		// reset
+		modelMap.value = {};
+		codeText.value = '# The following <variable>: <model> mappings are available\n';
```

Comment:
Whenever this watch is hit (when we open drill down) we will rewrite this codeText value
I do not see this code being saved in state right now but we will likely want to keep it later?

Approving as this is not an issue while we do not have this saved in state

Reply:
Code fragment is not saved right now. I think we might be able to just update the "info" section - but need to think about this a bit.